addons/stock_picking_batch/wizard/stock_add_to_wave.py

```diff
@@ -37,6 +37,7 @@ def default_get(self, fields_list):
     def attach_pickings(self):
         self.ensure_one()
 
+        self.env.context = dict(self.env.context, active_owner_id=self.user_id.id)
```

Comment:
`self = self.with_context(active_owner_id=self.user_id.id)`, no ?

Reply:
you're right. I thought It will be replaced by mine at first but it's merged